release-notes/v5.5.0.md

```diff
@@ -119,3 +119,7 @@
 #### <sub><sup><a name="v550-note-29" href="#v550-note-29">:link:</a></sup></sub> fix
 
 * Scrollbars have been given a touch of the Concourse design language. It should no longer use browser defaults. #4355
+
+#### <sub><sup><a name="v550-note-30" href="#v550-note-30">:link:</a></sup></sub> fix
```

Comment:
One final nit. Can you call this a `feature` rather than a `fix`? Folks have never had this before!

Reply:
Updated. I'm used to different nomenclature and rules - I was interpreting this as non-visible or difficult-to-keep-track-of step headers being 'fixed' so that they're now visible or easy to keep track of.

Thanks for the correction!